### ISSUE TITLE
Bug: package org.fxmisc.undo is not accessible in Java 11

### ISSUE THREAD

**Issue by achjaj** [FXMisc/RichTextFX#1008]
## Expected Behavior

Calling `getUndoManager().atMarkedPositionProperty()` on `org.fxmisc.richtext.CodeArea` object should return `ObservableBooleanValue atMarkedPositionProperty`

## Actual Behavior
Java compiler returns an error: 
````Error:(31, 40) java: org.fxmisc.undo.UndoManager.atMarkedPositionProperty() in package org.fxmisc.undo is not accessible
  (package org.fxmisc.undo is declared in the unnamed module, but module org.fxmisc.undo does not read it)
````

## Reproducible Demo

Provide a demo that maintainers of this project can copy, paste, and run to reproduce it immediately.

Use the following template to get started.
````java
public class Bug extends Application {

 public void start(Stage primaryStage) {
  CodeArea editor = new CodeArea();
  ObservableBooleanValue value = editor.getUndoManager().atMarkedPositionProperty(); // compiler should complaint here

  VBox parent = new VBox(editor);
  primaryStage.setScene(new Scene(parent));
  primaryStage.show();
 }

}
````

## Environment info:

- RichTextFX Version: 0.10.6
- Operating System: Arch Linux
- Java version: 11.0.10 (OpenJDK)
- I use Maven as my build system: [pom.xml](https://github.com/achjaj/Ju/blob/master/pom.xml)

<hr>

## Current Workarounds

None that I know about, except for using the lower java version.

## Implementation Costs

Open or export `org.fxmisc.undo` in `module-info.java` should work I think.

**Comment by Jugen** [comment]
The problem, I think, is that **undofx** is on the class path and not on the module path.
One way to fix this would be to edit the manifest file of undofx and add something like:
```
Automatic-Module-Name: org.fxmisc.undo
```
See attached file where I have updated it for you, just rename the zip to jar: [undofx-2.1.0.zip](https://github.com/FXMisc/RichTextFX/files/6177052/undofx-2.1.0.zip)

Alternatively you can use the [richtextfx-fat-0.10.6.jar](https://github.com/FXMisc/RichTextFX/releases/download/v0.10.6/richtextfx-fat-0.10.6.jar) instead, which includes all the dependencies and they will therefore all be on the module path automatically.

**Comment by achjaj** [comment]
I see. Do you think that a similar workaround as for #902 using Moditect could be possible here? Thing is, I would like to use only maven dependencies.

**Comment by Jugen** [comment]
I don't know about Moditect, best is to try it out and see .... (If you do, please give feedback.)

For the the future I'm going to try and publish a release for an **undofx** update with an automatic module in the manifest.
If I can get that right then it'll be used in the next release of RichTextFX and maven will sort it out for you.

**Comment by Jugen** [comment]
Okay, I've managed to update UndoFX with an automatic module in the jar manifest, and published version 2.1.1 to maven.
The current snapshot of RichtextFX has also been updated to use it.
